workflow/react-project/src/pages/MainPage/index.jsx

```diff
@@ -1,4 +1,9 @@
 import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { changeModalKey, deleteRandomCocktailInformation } from '../../store';
+import fetchRandomCocktail from '../../store/randomCocktail/asyncThunks';
```

Comment:
look here we see 3 action creators, but all of these are taken from different places. It's now useful because you alway have to remember whether action creator is in middleware or from store.
Lets create `/store/randomCocktail/actions.js` and import/export action creators connected to random cocktail here.
The same for modal stuff